### DISCUSSION TITLE
Manually specifying train and test data

### DISCUSSION THREAD

**Discussion by TashowarTanzim** [pycaret/pycaret#3542] [Q&A]
Hello everyone, so I have manually splitted my data and created a training and testing set (as per a stratifiedgroupkfold split I want). I want the model to be trained completely on my training data, ensuring that it is further not going to split it into a test-validation split (which might ruin the stratifiedgroupkfold split I have made). Now I am wondering, if I specify the parameters of the setup() function as such: demo = setup(data = training_data, test_data = testing_data, target = 'final_week_of_listing',........), will it work? Or do I have to specify the parameter "train_size = 1" to ensure that the training_data specified under "data" is not further splitted. If a similar discussion took place on another thread, please redirect me there, as I couldn't find such a topic. Thanks in advance!!

**Comment by michellebont** [comment]
Hi! I have the same problem. Have you found the solution?
